src/index.html

```diff
@@ -5,8 +5,441 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin ="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="icon">
+        <a href="#Home">
+          <img src="images/logo.png" alt="logomayo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+                class="is-active nav__link"
+                href="#APPLE"
+              >
+                Apple
+            </a>
+          </li>
```

Comment:
checklist 2(empty lines between), check your <a> (Each level of nesting, including text, contained within the element, requires 2-space offset.)

Reply:
agree, check your code for indents